cowprotocol/watch-tower
# chore: metrics todo and naming

# Description
This PR addresses issues around naming conventions for the prometheus metrics. Best efforts have been followed to adhere to [best practices](https://prometheus.io/docs/practices/naming/).

# Changes

- [x] Revised metric naming and labeling to be in accordance with best practices.
- [x] Removed custom API prometheus metrics and replaced with middleware.

## How to test

1. Run a sync from contract genesis.
2. Observe via `http://127.0.0.1:8080/metrics` there respective metrics changing.

## Related Issues

Related #78, #70
